signup.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
+    <title>login</title>
+</head>
+<body>
+    <div class="logo_box">
+        <img class="logo" src="images/login/logo.png">
+        <div class="login_box">
+            <div id="email">
+                <span class="email">이메일</span>
+                <h2 class="email_input"></h2>
+            </div>
+            <div id="nickname">
+                <span class="nickname">닉네임</span>
+                <h2 class="nickname_input"></h2>
+            </div>
+            <div id="password">
+                <span class="password">비밀번호</span>
+                <h2 class="password_input"></h2>
+            </div>
+            <div id="password_check">
+                <span class="password_check">비밀번호 확인</span>
+                <h2 class="password_check_input"></h2>
+            </div>
+            <h3 class="signup_button">
+                회원가입
+            </h3>
+            <div id="easy_login">
+                <div class="easy_login_box">
+                    <h3 class="easy_login">간편 로그인하기</h3>   
+                    <div class="login_sns">
+                        <a href="https://www.google.com/"
+                        target="_blank">
+                        <img class="login_google" src="images/login/google.png">
+                        </a>
```

Comment:
💊 제안

현재 코드에서는 하나의 태그가 나눠져 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
                        <a href="https://www.google.com/" target="_blank">
                          <img class="login_google" src="images/login/google.png">
                        </a>
```